README.md

```diff
@@ -4,7 +4,10 @@
 [![Coverage Status](https://coveralls.io/repos/github/trufflesuite/ganache-core/badge.svg?branch=develop)](https://coveralls.io/github/trufflesuite/ganache-core?branch=develop)
 # Ganache Core
 
-This is the core code that powers the Ganache application and the the Ganache command line tool.
+[Usage](#usage) | [Options](#options) | [Implemented Methods](#implemented-methods) | [Custom Methods](#custom-methods) | [Unsupported Methods](#unsupported-methods) | [Testing](#testing)
+--- | --- | --- | --- | --- | ---
+
+This is the core code that powers the Ganache application and the Ganache command line tool.
 
 # INSTALLATION
```

Comment:
While we're making things purdy, what do you think about making these subheadings 2nd level headings (by using `## Installation` instead of `# Installation`)? That and switching to Title Case instead of UPPER CASE?

Reply:
Sounds good to me!